Frontend.Integration.Tests/Pages/Projects/ProjectAssignment/ProjectAssignmentTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using AngleSharp.Dom;
+using AngleSharp.Html.Dom;
+using FluentAssertions;
+using Frontend.Models;
+using Newtonsoft.Json;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Frontend.Integration.Tests.Pages.Projects.ProjectAssignment
+{
+	public class ProjectAssignmentTests : BaseIntegrationTests
+	{
+		private readonly IntegrationTestingWebApplicationFactory _factory;
+
+		public ProjectAssignmentTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
+		{
+			_factory = factory;
+		}
+
+		[Fact]
+		public async Task Should_unassign_a_project()
+		{
+			var project = GetProject();
+			_factory.AddAnyPatch($"/academyTransferProject/{project.ProjectUrn}", project);
+
+			await OpenUrlAsync($"/project-assignment/{project.ProjectUrn}");
+
+			Document.QuerySelector<IHtmlInputElement>("#UnassignDeliveryOfficer").Value = "true";
+			await Document.QuerySelector<IHtmlFormElement>("form").SubmitAsync();
+
+			Document.Url.Should().EndWith($"project/{project.ProjectUrn}");
```

Comment:
Might be worth having an assertion that the field reads as Empty. To confirm it's not just moved pages but updated the AssignedUser value on the UI.